themes/bootstrap5/scss/mixin-overrides/color-mode.scss

```diff
@@ -0,0 +1,25 @@
+/* Override of built-in color mode, to output both types */
+
+@mixin color-mode($mode: light, $root: false) {
+  // Generally ignore $color-mode-type and output handlers for both types.
+  // Handle media-query $color-mode-type
+  @if $root == true {
+    @media (prefers-color-scheme: $mode) {
+      :root:not([data-bs-theme]) {
```

Comment:
This `:not` is so that the media query (system-defined) color scheme never overrides a DOM-defined color scheme.